test-it/ViaTests/TestVI_VariantToData_DefaultValueOfTypeTerminalIsWrittenIntoOutputInCaseOfError.via

```diff
@@ -0,0 +1,244 @@
+// Autogenerated Vireo assembly file
+// Generated on 04/05/2019 12:40:43
+//---------------------------------------------------
+//VI Definition: ::Web Server::Interactive::Application::TestVI_VariantToData_DefaultValueOfTypeTerminalIsWrittenIntoOutputInCaseOfError.gviweb
```

Comment:
![case1](https://user-images.githubusercontent.com/2609749/55647080-6312b680-57a2-11e9-85f4-6062e89ba9f4.PNG)
![case2](https://user-images.githubusercontent.com/2609749/55647088-660da700-57a2-11e9-8bf8-89e1c6050f10.PNG)

Reply:
The last string tunnel on the right is a keep-last-value.